### ISSUE TITLE
Declare dependencies to allow portability.

### ISSUE THREAD

**Issue by Kreijstal** [PPierzc/colab-component#1]
It'd be nice if the widgets were also portable, check this out https://github.com/blois/jupyter-output-spec

**Comment by PPierzc** [comment]
True, there are a lot of different notebook editors out there. Making the components independent of the editor would be a nice addition. I will add this as a milestone for this project
